frontend/packages/shared-components/src/Svg.stories.js

```diff
@@ -0,0 +1,50 @@
+import { Svg } from './';
+
+export default {
+  title: 'Shared Components/Svg',
+  component: Svg,
+  argTypes: {
+    name: {
+      control: {
+        type: 'select',
+      },
+      options: [
+        'Active',
+        'AngleDown',
+        'AngleUp',
+        'ArrowLeft',
+        'ArrowLeftBold',
+        'ArrowRight',
+        'Bin',
+        'Calendar',
+        'CaretDown',
+        'CheckCircle',
+        'CheckMark',
+        'Close',
+        'Copy',
+        'Discord',
+        'Eye',
+        'GitHub',
+        'HideEye',
+        'Image',
+        'InfoOutLine',
+        'Instagram',
+        'InvalidCheckMark',
+        'LinkOut',
+        'Logo',
+        'Plus',
+        'Star',
+        'Twitter',
+        'Upload',
+        'ValidCheckMark',
+        'Website',
+      ],
+    },
+  },
+};
+
+const Template = (args) => <Svg {...args} />;
+export const Controls = Template.bind({});
+Controls.args = {
+  name: 'Eye',
+};
```

Comment:
Should we have this file inside the Svg folder? So each shared component has it's own `.stories` file and on a high level we have folders only. Also for updating the .stories file and at the same time the component structure, might be easier and closer